crates/orderbook/src/database/solver_competition.rs

```diff
@@ -97,4 +105,78 @@ mod tests {
         let result = db.load(id + 1).await.unwrap_err();
         assert!(matches!(result, LoadSolverCompetitionError::NotFound(_)));
     }
+
+    #[tokio::test]
+    #[ignore]
+    async fn temp_test_migration() {
+        let db = Postgres::new("postgresql://").unwrap();
+        database::clear_DANGER(&db.pool).await.unwrap();
+
+        db.save(SolverCompetition {
+            gas_price: 1.,
+            transaction_hash: Some(H256([5; 32])),
+            ..Default::default()
+        })
+        .await
+        .unwrap();
+
+        db.save(SolverCompetition {
+            gas_price: 2.,
+            transaction_hash: None,
+            ..Default::default()
+        })
+        .await
+        .unwrap();
+
+        #[derive(Debug, sqlx::FromRow)]
+        struct Row {
+            id: i64,
+            json: sqlx::types::JsonValue,
+        }
+
+        let rows: Vec<Row> = sqlx::query_as("SELECT * FROM solver_competitions")
+            .fetch_all(&db.pool)
+            .await
+            .unwrap();
+
+        for row in rows {
+            println!("{:#?}", row);
+        }
+
+        // ---
+
+        let mut transaction = db.pool.begin().await.unwrap();
+        transaction
+            .execute(
+                r#"
+ALTER TABLE solver_competitions
```

Comment:
Really cool test! great.

Uber nit Nit: 
Maybe you could have used the content from the file: database/sql/V025__settlement_competition_tx_hash.sql directly instead of rewriting it here.